pkg/pillar/zedcloud/send.go

```diff
@@ -86,9 +86,7 @@ func SendOnAllIntf(ctx ZedCloudContext, url string, reqlen int64, b *bytes.Buffe
 		}
 		for _, intf := range intfs {
 			resp, contents, rtf, err := SendOnIntf(ctx, url, intf, reqlen, b, allowProxy)
-			if rtf {
-				remoteTemporaryFailure = true
-			}
+			remoteTemporaryFailure = rtf
```

Comment:
No, you need to preserve the logic to only set this if rtf is not the default (which was "false" and now is the 0th enum value "None").

Reply:
ok. will check.

Reply:
i think the logic is preserved. inside SendOnIntf(), the place before receives rtf=true is converted to types.SenderStatusRemTempFail, and all the places calling SendOnAllIntf(), now will explicitly checks for if rtf == types.SenderStatusRemTempFail.

Reply:
You're in a loop. The original code sets the rtf boolean if any SendOnIntf returns it. Your code sets it based on the last iteration of the loop.

Reply:
Ha, got it.